libnmstate/__init__.py

```diff
@@ -15,6 +15,9 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 from __future__ import absolute_import
+from libnmstate._error import NmstateError
```

Comment:
- I do not understand and like private modules in general and `error` is usually public.
- There is also a separation between import module types using blocks, this new import is equivalent to the validator one, please keep it in that block.
- It is preferred to expose modules and not specific components in them, see validator.